numpy/_core/code_generators/ufunc_docstrings.py

```diff
@@ -4275,6 +4279,135 @@ def add_newdoc(place, name, doc):
     See Also
     --------
     str.isalpha
+
+    """)
+
+add_newdoc('numpy._core.umath', 'isdigit',
+    """
+    Returns true for each element if all characters in the string are
+    digits and there is at least one character, false otherwise.
+
+    For 8-bit strings, digits are the byte values in the sequence
+    b'0123456789'. For Unicode strings, digits include decimal
+    characters and digits that need special handling, such as the
+    compatibility superscript digits. This also covers digits which
+    cannot be used to form numbers in base 10, like the Kharosthi numbers.
+
+    Parameters
+    ----------
+    x : array_like, with ``bytes_`` or ``unicode_`` dtype
+    $PARAMS
+
+    Returns
+    -------
+    y : ndarray
+        Output array of bools
+        $OUT_SCALAR_1
+
+    See Also
+    --------
+    str.isdigit
+
+    Examples
+    --------
+    >>> a = np.array(['a', 'b', '0'])
+    >>> np.char.isdigit(a)
+    array([False, False,  True])
+    >>> a = np.array([['a', 'b', '0'], ['c', '1', '2']])
+    >>> np.char.isdigit(a)
+    array([[False, False,  True], [False,  True,  True]])
+
+    """)
+
+add_newdoc('numpy._core.umath', 'isspace',
+    r"""
+    Returns true for each element if there are only whitespace
+    characters in the string and there is at least one character,
+    false otherwise.
+
+    For 8-bit strings, whitespace characters are the ones in the
```

Comment:
Minor nit, instead `8-bit strings` here and elsewhere, which is a remnant of Python2 I think, just refer to it as a `bytes_` dtype or `byte strings`.